balboa-client/src/main/scala/com/socrata/metrics/collection/PreBufferQueues.scala

```diff
@@ -1,13 +1,13 @@
 package com.socrata.metrics.collection
 
-import java.util.concurrent.LinkedBlockingQueue
+import java.util.concurrent.{BlockingQueue, LinkedBlockingQueue}
 
 import com.socrata.metrics.components.MetricEntry
 
 trait PreBufferQueue {
-  def queue:SocrataAbstractQueue[MetricEntry]
+  def queue: BlockingQueue[MetricEntry]
```

Comment:
Square-ish worlds would make the square whee only slightly less suboptimal. Now if you had a world where the vector of the gravitational acceleration was changing 90 degrees at the proper frequency, perhaps you'd have a point. Even then, the performance of square wheel vs. circular wheel would be about the same.